### DISCUSSION TITLE
"Not Ready. Please check your connection" on fresh install of RustDesk client on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by matthewmturner** [rustdesk/rustdesk#7085] [Q&A] [answered]
Basically as simple as the title suggests. I have brand new Ubuntu 22 machine and just installed Rustdesk but getting "Not Ready. Please check your connection".  Not self hosting, just trying to use open source / free for now.  Ive made sure everything is up to date on the machine and restarted several times but still no luck.

**Comment by matthewmturner** [comment]
Also just tried on Mac and same issue.

**Reply by rustdesk** [reply, marked as answer]
https://twitter.com/rustdesk/status/1754818959637942671

**Reply by matthewmturner** [reply]
@rustdesk thank you for the information and sorry to hear your are going through this. wishing the best to you and the team
